openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -57,47 +58,50 @@ class Meta(object):
     display_org_with_default = TextField()
 
     # Start/end dates
-    start = DateTimeField(null=True)
-    end = DateTimeField(null=True)
-    advertised_start = TextField(null=True)
-    announcement = DateTimeField(null=True)
+    start = DateTimeField(blank=True, null=True)
```

Comment:
This was done to have opportunity to change CourseOverviews manually from admin.